package.json

```diff
@@ -92,6 +92,7 @@
     "ethereum-cryptography": "^1.0.3",
     "isomorphic-fetch": "^3.0.0",
     "json-bigint": "^1.0.0",
+    "json-keys-sort": "^2.1.0",
```

Comment:
```
const ordered = Object.keys(unordered).sort().reduce(
  (obj, key) => { 
    obj[key] = unordered[key]; 
    return obj;
  }, 
  {}
);
```

should we replace the dependency by a simple util function?

Reply:
I think this function will not do deep sorting, only top-level keys. I looked into this package source it has one file with validation and deep sort.